src/V8JsiRuntime.cpp

```diff
@@ -1574,12 +1572,54 @@ v8::Local<v8::Value> V8Runtime::valueRef(const jsi::Value &value) {
   }
 }
 
-std::unique_ptr<jsi::Runtime> makeV8Runtime(V8RuntimeArgs &&args) {
+void applyOverrides(V8RuntimeArgs& args) {

```

Comment:
>applyOverrides [](start = 5, length = 14)

It would be great to avoid this DLL accessing the registry. Can it be the application that does it?
The V8JSI is an open source project and it seems to be wrong to use MS Office key in the path and have a "backdoor" affecting any app in the system that uses this DLL.

Reply:
I do partially agree Vladimir .. but having such control will be extremely helpful when building debugging and performance tooling. We can hopefully remove these once our tooling is matured. I'm removing the "Office" subpaths from the registry key.. Hope it works for you.

Reply:
Can we have a subkey per application? I.e. per executable file name that uses the V8JSI DLL.

---
In reply to: [606578849](https://github.com/microsoft/v8-jsi/pull/50#discussion_r606578849) [](ancestors = 606578849)

Reply:
Why is this needed at all? These values are already being passed through the arguments; the consumer is already capable of passing them in.

---
In reply to: [606610175](https://github.com/microsoft/v8-jsi/pull/50#discussion_r606610175) [](ancestors = 606610175,606578849)